### PR TITLE
Add index checking to workflow

### PR DESCRIPTION
**Pull request by johandahlberg**
Added index checking to the workflow (related PR at siswrap here: https://github.com/arteria-project/arteria-siswrap/pull/16).

In addition to this I fixed a error in the README, removed a unused config value, and refactored the `poll_status` script to use the st2 method of logging and returning for exit codes.

### CODE REVIEW

File: actions/workflows/ngi_uu_workflow.yaml [hunk 1]
```diff
@@ -177,9 +177,32 @@ workflows:
                 input:
                     url: "<% $.report_status_url %>"
                 on-success:
-                    - download_qc_config
+                    - run_sisyphus_check_indicies
             ### QUICK REPORT END ###
 
+            ### CHECK INDICIES START###
```
Comment (johanherman) [inline actions/workflows/ngi_uu_workflow.yaml]: `Indices`, not `indicies`.

**Comment by johanherman** [comment]
I like the refactoring, but where will the logs via the st2 framework end up?

File: actions/workflows/ngi_uu_workflow.yaml (hunk 1 above)
Comment (johandahlberg) [inline actions/workflows/ngi_uu_workflow.yaml]: :+1:

**Comment by johandahlberg** [comment]
It ends up in the actions `stderr`, see this example:

```
id: 57c6c71e5249e52357abd331
status: succeeded (2403s elapsed)
parameters: 
  url: http://mm-xart002:10700/api/1.0/report/status/21508
result: 
  exit_code: 0
  result:
  - 0
  - host: mm-xart002
    msg: Process was completed successfully with return code 0.
    pid: 21508
    runfolder: /data/mm-xart002/runfolders//150821_M00485_0220_000000000-AG2UJ
    state: done
    stderr: 'Using random seed 822 based on fcId 000000000-AG2UJ

      '
    stdout: ''
  stderr: 'st2.actions.python.PollStatus: INFO     2016-08-31 14:01:35.167774 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:02:35.240850 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:03:35.253228 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:04:35.318701 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:05:35.386708 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:06:35.420897 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:07:35.488957 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:08:35.512463 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:09:35.552026 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:10:35.618719 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:11:35.669872 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:12:35.696899 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:13:35.746762 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:14:35.787500 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:15:35.854666 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:16:35.904498 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:17:35.915538 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:18:35.932137 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:19:35.983438 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:20:36.050297 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:21:36.116798 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:22:36.186147 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:23:36.254665 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:24:36.318984 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:25:36.385438 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:26:36.436638 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:27:36.448764 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:28:36.484431 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:29:36.529022 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:30:36.595655 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:31:36.666016 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:32:36.734671 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:33:36.769032 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:34:36.824679 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:35:36.880815 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:36:36.952019 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:37:37.012435 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:38:37.030835 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:39:37.098497 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:40:37.166683 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state started. Sleeping 1m until retrying again...

    st2.actions.python.PollStatus: INFO     2016-08-31 14:41:37.233115 -- http://mm-xart002:10700/api/1.0/report/status/21508 returned state done. Will now stop polling the status.

    '
  stdout: ''

```

File: actions/workflows/ngi_uu_workflow.yaml [hunk 2]
```diff
@@ -180,7 +180,7 @@ workflows:
                     - run_sisyphus_check_indicies
             ### QUICK REPORT END ###
 
-            ### CHECK INDICIES START###
+            ### CHECK INDICES START###
```
Comment (johanherman) [inline actions/workflows/ngi_uu_workflow.yaml]: I'm sorry, I should've been more clear. You have this typo in many places in the workflow. :)

**Comment by johanherman** [comment]
Hm, okay. Perhaps a bit weird to have it in `stderr`, but it is not a big deal. More seriously though - is it possible to tail this log or do we then have to wait for the task to finish before we can debug/troubleshoot the output?

**Comment by johandahlberg** [comment]
Nope, you have to wait for the task to finish, at least as far as I know. I have a vague memory of seeing some feature request about it on the st2 repo, but I'm not sure.

**Comment by johanherman** [comment]
That's a bit cumbersome IMHO. Do you really think it is acceptable during development? Seems like it will/can increase the dev-test cycle. Is it possible to force the output to some other backend (e.g. during development)?

**Comment by johandahlberg** [comment]
Here are the docs https://docs.stackstorm.com/actions.html#logging Since it is a standard python logger I guess we might be able to set it up to multiple stream, directing one to stderr and another to file? I haven't tried it though.

Also I've never locked at the logs while the action execute, so even if I see you point it hasn't effected me much...

**Comment by johanherman** [comment]
Ok. Guess we can run with this for now then. Might be able to do as you suggest later if necessary.

**Pull request merged**